src/se/academy/wumpus/Game.java

```diff
@@ -40,6 +42,10 @@ public boolean isOver() {
     }
 
     public static void main(String args[]) {
+        char U = '\u033F';
+        System.out.println("Hunt the Wumpus\n" + "         "+U+U+U+U+U+U+"\n" );
```

Comment:
That's good, but consider doing this in the constructor of `Game` instead, using `output`. That way, if someone wanted to, they could test that the welcome message is there.